cobaya/samplers/polychord/polychord.py

```diff
@@ -180,7 +180,7 @@ def initialize(self):
         locs = bounds[:, 0]
         scales = bounds[:, 1] - bounds[:, 0]
         # This function re-scales the parameters AND puts them in the right order
-        self.pc_prior = lambda x: (locs + np.array(x)[self.ordering] * scales).tolist()
+        self.pc_prior = lambda x: [self.model.prior.pdf[i].ppf(xi) for i, xi in enumerate(np.array(x)[self.ordering])]
```

Comment:
The `pdf.ppf` thing might read a bit confusing. The thing is that `prior.pdf` is really a `scipy.stats.rv_continuous` class that has `.pdf`, `.logpdf` and `.ppf` as functions. Might be worth considering to rename `prior.pdf` to something else? (e.g. `prior.rv_continuous` or `prior.distributions`?)